paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/DataCatalogTable.java

```diff
@@ -50,6 +54,17 @@ public Table table() {
         return table;
     }
 
+    @Override
+    public Schema getUnresolvedSchema() {
+        // add physical column comments
+        Map<String, String> columnComments =
+                table.rowType().getFields().stream()
+                        .filter(dataField -> dataField.description() != null)
+                        .collect(Collectors.toMap(DataField::name, DataField::description));
+
+        return FlinkTableSchemaUtils.toSchema(getSchema(), columnComments);
```

Comment:
Put `FlinkTableSchemaUtils.toSchema` to `DataCatalogTable.toSchema`? No other use it.